test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -160,15 +161,33 @@ function copyManifestToNodes() {
 FLOW_AGG_YML="/tmp/flow-aggregator.yml"
 SAVED_FLOW_AGG_IMG=/tmp/flow-aggregator.tar
 FLOW_AGG_IMG_NAME=projects.registry.vmware.com/antrea/flow-aggregator:latest
+
+CH_OPERATOR_INSTALL_BUNDLE_BASE_YML=$THIS_DIR/../../../../build/yamls/clickhouse-operator-install-bundle.yml
```

Comment:
nit: `CH_OPERATOR_INSTALL_BUNDLE_BASE_YML` -> `CH_OPERATOR_INSTALL_BUNDLE_YML`
We usually don't call it a base yml except it locates under `build/yamls/base`

Reply:
I see. Should I also change the name of this one ANTREA_BASE_YML?

Reply:
We could keep `ANTREA_BASE_YML` IMO, since we will generate an `ANTREA_YML`, it will help us distinguish them.

Reply:
Ok, fixed